public/background.js

```diff
@@ -0,0 +1,40 @@
+const EXTENSION_ENABLED_KEY = "settings:extension_enabled"
```

Comment:
@Ohnoimded thank you for this awesome PR 🙌 

This feature was requested many times on Github and in extension reviews, I'm glad you're on it! ❤️ 

I have a couple of questions/suggestions, let's chat and then merge this PR?